nomenclature/codelist.py

```diff
@@ -191,22 +191,26 @@ def from_directory(
         instance of cls (:class:`CodeList` if not inherited)
 
         """
-        code_list = cls._parse_codelist_dir(path, file_glob_pattern)
 
-        if config is not None:
+        if (
```

Comment:
Maybe a try-except is the more pythonic approach here?

Reply:
Ah yes of course, thanks. I couldn't think of that one...